scripts/check_release_branch_date.py

```diff
@@ -0,0 +1,27 @@
+import json
+from datetime import datetime, date, timedelta
+from urllib import request
+
+from common import get_patch_version, env
+
+if __name__ == '__main__':
+    repo = env("GITHUB_REPOSITORY")
+    response = request.urlopen(f"https://api.github.com/repos/{repo}/milestones")
+    milestones = json.load(response)
+
+    milestone_version = f"v{get_patch_version()}"
+    milestone = next(milestone for milestone in milestones if milestone["title"] == milestone_version)
+    # TODO: find out more correct way to parse data
+    release_date = datetime.strptime(milestone["due_on"], "%Y-%m-%dT%H:%M:%SZ").date()
```

Comment:
Maybe we can use [datetime.date.fromisoformat](https://docs.python.org/3/library/datetime.html#datetime.date.fromisoformat) (Python 3.7+) here to parse data

Reply:
I tried it but it doesn't work. I think it's because it supports only `YYYY-MM-DD` format

Reply:
Oops, wrong one, [datetime.datetime.fromisoformat](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat).
It fails to work with timezone (`Z` at the end), but without it is ok.